src/actions/ActionHelpers.ts

```diff
@@ -0,0 +1,664 @@
+// eslint-disable-next-line import/no-unresolved
+import { Frame, FrameType, IHasName, Layout, ListVariable, Page, StylekitChacterStylesController, StylekitColorsController, StylekitParagraphStylesController, Variable, VariableValue } from 'grafx-studio-actions';
+
+/**
+ * Whether the variable is the trigger for this action
+ * @param name optional variable name
+ * @returns true if it was the trigger, otherwise false
+ */
+export function isVariableTriggered(name?: string | Variable): boolean {
```

Comment:
@Matthiee It seems like name should not be optional, because if you do not provide a name you will always get an error

Reply:
@seancrowe the implementation is incorrect, I will fix it. When providing it without any arguments it will return true for any variable. When providing it with a name, it will only return true if it is both a variable trigger and the name matches.

Reply:
Remove